frontend/src/scenes/notebooks/Nodes/NotebookNodeBacklink.tsx

```diff
@@ -0,0 +1,87 @@
+import { mergeAttributes, Node, NodeViewProps } from '@tiptap/core'
+import { NodeViewWrapper, ReactNodeViewRenderer } from '@tiptap/react'
+import { InsightModel, NotebookNodeType } from '~/types'
+import { Link } from '@posthog/lemon-ui'
+import { IconGauge, IconBarChart, IconFlag, IconExperiment, IconLive, IconPerson, IconCohort } from 'lib/lemon-ui/icons'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { urls } from 'scenes/urls'
+import { useActions, useValues } from 'kea'
+import { notebookSidebarLogic } from '../Notebook/notebookSidebarLogic'
+import { notebookLogic } from '../Notebook/notebookLogic'
+
+const ICON_MAP = {
+    dashboards: <IconGauge />,
+    insights: <IconBarChart />,
+    feature_flags: <IconFlag />,
+    experiments: <IconExperiment />,
+    events: <IconLive />,
+    persons: <IconPerson />,
+    cohorts: <IconCohort />,
+}
```

Comment:
Tried it locally and the icons are weirdly placed 

<img width="215" alt="Screenshot 2023-07-14 at 16 19 46" src="https://github.com/PostHog/posthog/assets/2536520/2174cc4c-51c3-4ba8-9aef-e9cb8a41d7f7">